README.md

```diff
@@ -5,4 +5,30 @@
 
 # Google Container Tools for IntelliJ
 
-In development
+This plugin adds support for [Kubernetes](https://www.kubernetes.io) development into [JetBrains family of IDEs](https://www.jetbrains.com/products.html), including IntelliJ (both Community and Ultimate editions), GoLand, PyCharm, WebStorm (and others).
```

Comment:
don't forget the **the** (..development into the* JetBrains family of ..)

Reply:
Oops, added. Thanks!